tests/drivers/fprotect/positive/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+target_include_directories(app PRIVATE .)
+target_include_directories(app PRIVATE ${ZEPHYR_BASE}/../nrf/subsys/bootloader/)
```

Comment:
Are these two lines actually needed? The test app does not seem to be including anything from these locations.

Reply:
Yep, this shouldn't be needed any more.

Reply:
Done